examples/AerosolActivation/aerosol_activation.jl

```diff
@@ -0,0 +1,275 @@
+# # Aerosol Activation Example
+
+# ## Overview
+
+# This example is based on AerosolActivation module which is a part of the
+#   [CloudMicrophysics.jl](https://github.com/CliMA/CloudMicrophysics.jl) package.
+# The AerosolActivation module computes the total number and mass
+#   of aerosol particles that get activated and become cloud droplets,
+#   given the atmospheric conditions and
+#   the initial aerosol size distribution and properties.
+# See the AerosolActivation module
+#   [docs](https://clima.github.io/CloudMicrophysics.jl/dev/Aerosol_activation/)
+#   for derivation and description of all input parameters.
+#
+# In this example we use the ensemble Kalman methods to learn
+#   two parameters that describe the chemical composition of aerosol particles
+#   based on the observed total number and mass of activated particles.
+# The AerosolActivation model is used here in a "perfect model" setting,
+#   meaning that the observations are generated by the same module
+#   we are calibrating.
+
+# ## Prerequisites
+
+# The example depends on some standard Julia libraries,
+#   as well as the CliMA packages:
+#   [EnsembleKalmanProcess.jl](https://github.com/CliMA/EnsembleKalmanProcesses.jl),
+#   [CLIMAParameters.jl](https://github.com/CliMA/CLIMAParameters.jl) and
+#   [CloudMicrophysics.jl](https://github.com/CliMA/CloudMicrophysics.jl).
+# To ensure that all the dependencies are met
+#   start Julia using the Project.toml file provided in the example
+#   and run the Julia package manager to download all the dependecies.
+
+# ## Example
+
+# We begin by importing some standard Julia modules,
+# the Ensemble Kalman Process modules,
+# CLIMA Parameter modules and Aerosol Activation modules.
+using Plots
+using Distributions
+using LinearAlgebra
+using Test
+using Random
+rng_seed = 44
+
+import EnsembleKalmanProcesses
+const EKP = EnsembleKalmanProcesses
+
+import CLIMAParameters
```

Comment:
```suggestion
using CLIMAParameters
```

Reply:
I think it's a matter of style preference? I was taught to use `import` when bringing in our packages. I think it makes it clearer from where individual functions and parameters are coming from. The CloudMicrophysics.jl package tests first were first written using `using`:

https://github.com/CliMA/CloudMicrophysics.jl/pull/26/files

I think in that PR I went overboard and was doing it for even the standard packages like `Plots` or `Tests` :) Since then I settled on using `using` for standard packages and `import` for our developed packages.

Reply:
I disagree it's just a style.

`using A` will bring `A` and make all exports of `A` available in namespace while with `import B` you still have to call `B.anicefunctionwithinB` even if `anicefunctionwithinB` is exported.

Reply:
Right, I agree. I guess I just prefer to call it `B.anicefunctionwithinB` because it forces me to know that my function comes from `B`. Between the `Thermodynamics.jl` `CloudMicrophysics.jl` and `CLIMAParameters.jl` is sometimes not so easy to guess who comes from where.

However if this is not the style of `EKP.jl` I can change this. It's just funny because not too long ago someone convinced me I should change it the other way.

Reply:
No leave it as is! I see your point now! :)